### PR TITLE
Demo ui changes

### PR DESCRIPTION
**Pull request by surekhag**

### CODE REVIEW

File: client/src/components/DeleteModal/deleteModal.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import GridItem from '../../components/Grid/GridItem'
+
+export const DeleteModal = props => {
+  const { title, showDelDialog, handleYesDelete, handleNoDelete } = props
+
+  return (
+    <Dialog title={title} modal={true} open={showDelDialog}>
+      <DialogActions>
+        <GridItem xs={12} sm={12} md={12}>
+          <p> Are you sure you want to delete this Information ? </p>
```
Comment (bhushan-oe) [inline client/src/components/DeleteModal/deleteModal.js]: make this text also dynamic through props

File: client/src/components/DeleteModal/deleteModal.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { Dialog, DialogActions } from '@material-ui/core'
+import React from 'react'
+import Button from '../../components/CustomButtons/Button'
+// import styles from '../../assets/jss/material-dashboard-react/views/dashboardStyle'
+import GridItem from '../../components/Grid/GridItem'
+
+export const DeleteModal = props => {
```
Comment (bhushan-oe) [inline client/src/components/DeleteModal/deleteModal.js]: make it generic modal so, at any place we can use it...e.g. confirmation on edit, change in password etc.

File: client/src/components/DeleteModal/deleteModal.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  const { title, showDelDialog, handleYesDelete, handleNoDelete } = props
+
+  return (
+    <Dialog title={title} modal={true} open={showDelDialog}>
+      <DialogActions>
+        <GridItem xs={12} sm={12} md={12}>
+          <p> Are you sure you want to delete this Information ? </p>
+          <div style={{ textAlign: 'center' }}>
+            {' '}
```
Comment (bhushan-oe) [inline client/src/components/DeleteModal/deleteModal.js]: remove extra space

File: client/src/components/DeleteModal/deleteModal.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            <Button
```
Comment (bhushan-oe) [inline client/src/components/DeleteModal/deleteModal.js]: put condition so button will be visible if there is click handler for it.

File: client/src/layouts/User/User.js [hunk 5]
```diff
@@ -78,10 +78,7 @@ export default function User({ ...rest }) {
     }
   }
   React.useEffect(() => {
-    if (
-      currentUser &&
-      (currentUser.userRole == 'Admin' || currentUser.userRole == 'admin')
-    ) {
+    if (currentUser && currentUser.userRole === 'Admin') {
```
Comment (bhushan-oe) [inline client/src/layouts/User/User.js]: why are we keeping role as Capitalized name. keep it small case only.

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: client/src/components/DeleteModal/deleteModal.js (hunk 1 above)
Comment (surekhag) [reply client/src/components/DeleteModal/deleteModal.js]: Done

File: client/src/components/DeleteModal/deleteModal.js (hunk 2 above)
Comment (surekhag) [reply client/src/components/DeleteModal/deleteModal.js]: Done

File: client/src/components/DeleteModal/deleteModal.js (hunk 3 above)
Comment (surekhag) [reply client/src/components/DeleteModal/deleteModal.js]: Done

File: client/src/components/DeleteModal/deleteModal.js (hunk 4 above)
Comment (surekhag) [reply client/src/components/DeleteModal/deleteModal.js]: Done

File: client/src/layouts/User/User.js (hunk 5 above)
Comment (surekhag) [reply client/src/layouts/User/User.js]: All constant values used in EMS starts with capital letters only.. 'Admin' is one of those ...

Comment (surekhag) [reply client/src/layouts/User/User.js]: either all shall be small case or all shall be in upper came case .. please confirm ..

Comment (surekhag) [reply client/src/layouts/User/User.js]: Done. Also updated API for same

File: client/src/components/Modal/modal.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import { Dialog, DialogActions } from '@material-ui/core'
+import React from 'react'
+import Button from '../CustomButtons/Button'
+// import styles from '../../assets/jss/material-dashboard-react/views/dashboardStyle'
+import GridItem from '../Grid/GridItem'
+
+export const Modal = props => {
```
Comment (bhushan-oe) [inline client/src/components/Modal/modal.js]: Can you rename it to `AlertModal` ?

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

**Review by bhushan-oe — APPROVED** [review]

**Pull request merged**